shared/cfg/guest-hw.cfg

```diff
@@ -115,6 +115,10 @@ variants:
     - luks:
         image_format = luks
         image_secret = redhat
+        # image_secret_obj_file
```

Comment:
Considering that if there are other expressions of the secret data, bool values is not good to handle that, thus suggest using enum values, for example `image_secret_expression_stg1 = file; image_secret_expression_stg2 = data`

Reply:
OK, I will update it. Thanks.